scripts/iam_auto_deployment.sh

```diff
@@ -13,20 +13,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -o errexit
+
 # Preparation
 TRED='\e[91m'
 TYELLOW='\e[93m'
 TGREEN='\e[92m'
 TNC='\033[0m'
+LABEL_SAFE_TIMESTAMP=$(date --utc +%Ft%Tz | sed -e 's/:/-/g')
 # Set the toplevel forseti folder.
 if command -v git > /dev/null; then
 	repodir="$(git rev-parse --show-toplevel 2> /dev/null || echo "/home/$USER/forseti-security")"
 else
 	repodir="/home/$USER/forseti-security"
 fi
 
+# Config variables: the following variables are set during config and
+# are used to prepare, conifgure, or execute deployment. They are
+# pre-set here in order to document document the variables and to ensure
+# that they are in the shells global namespace.
+ROOTTYPE=""
+ROOTID=""
+SCRAPINGSA=""
+GSUITE_ADMINISTRATOR=""
+BRANCHNAME=""
+SQLINSTANCE=""
+PROJECT_ID=""
+DEPLOYMENTNAME=""
+
+# WARNING: THESE VALUES ARE NOT CONFIGURABLE!
+# These defaults match values contained in the example deployment YAMLs.
+# These should not be considered to be configurable but they are placed
+# here to avoid repetition.
+FORSETI_DB_NAME="forseti_security"
```

Comment:
This variable is no longer used. Now that we have combined the databases, I recommend just leaving in FORSETI_DB_NAME and replacing all uses of EXPLAIN_DB_NAME with FORSETI_DB_NAME.

Reply:
Fixed